ouroboros-consensus-cardano-test/src/Test/ThreadNet/Infra/ShelleyBasedHardFork.hs

```diff
@@ -237,12 +252,16 @@ protocolInfoShelleyBasedHardFork protocolParamsShelleyBased
     protocolInfo1 =
         protocolInfoShelleyBased
           protocolParamsShelleyBased
+          ()
```

Comment:
I've heard about "magic numbers". But this is the first time I'm encountering "magic units" :)
Maybe we should at least add comment `-- translation context` or a helper function `defaultTranslationContext :: TranslationContext era` that would equal to unit.

Reply:
Ha, yes. We talked about this before too.

I'll add comments in these applications where it's not obvious. We should also follow-up with Ledger, asking them to use a bespoke `NoTranslationContext` isomorph instead of `()`.

Reply:
I pushed up a fixup that adds comments to the bare occurrences of `()`.